### DISCUSSION TITLE
the freezed executable don't work

### DISCUSSION THREAD

**Discussion by Venoob** [marcelotduarte/cx_Freeze#929] [Q&A]
I just finished freezing my program. I have the main.exe in the build directory but when I run it ,it just stop without giving any error message even I run it with a commandline .
There is my freezing script:
```
import sys, os
from cx_Freeze import setup, Executable
PYTHON_INSTALL_DIR = os.path.dirname(os.path.dirname(os.__file__))

sys.path.append('pandastable')

includes = ["pandastable"]
includefiles = [
                os.path.join(PYTHON_INSTALL_DIR, 'DLLs', 'tk86t.dll'),
                os.path.join(PYTHON_INSTALL_DIR, 'DLLs', 'tcl86t.dll'),
                'icone.ico','VCbase.db','ClasseAgents.py','ClasseData.py','ClassePerformance.py','ClasseTime.py','ClasseTraitement.py','PredictionFlux.py'
                ]


build_exe_options = {"packages": ["os","numpy","matplotlib","pandas",'ClasseAgents.py','ClasseData.py','ClassePerformance.py','ClasseTime.py','ClasseTraitement.py','PredictionFlux.py',
                                  "scipy",
                                  "pandastable"],
                     "namespace_packages": ['mpl_toolkits'],
                     "include_msvcr": True,
                     "includes": includes,
                     "include_files": includefiles}

base = None
if sys.platform == "win32":
    base = "Win32GUI"

executables = [Executable("Main.py", base=base,
                          #copyDependentFiles = True,
                          targetName='TaskManager.exe',
                          shortcutName="TakManaer",
                          shortcutDir="DesktopFolder",
                          icon="icone.ico")]

setup(  name = "Task manager for BPO",
	version = "1.0",
	description = "task manager est un gestionnaire de traitement intelligent",
    options = {"build_exe": build_exe_options},
    executables = executables)
```

**Comment by marcelotduarte** [comment]
1) filename.py is not a package - so don't include this in packages and probably unnecessary in include_files;
2) tcl/tk is detected by cx_Freeze in Windows, if you're using the official python
3) try to use base=None only to test/debug and execute by command line to see if it reports some error.

**Reply by Venoob** [reply]
thank you, and what if I use Annaconda distribution

**Reply by marcelotduarte** [reply]
I'm not sure about anaconda.

**Reply by Venoob** [reply]
hello, I found the problem, scipy.special.cython_special was not found ,,,

**Reply by marcelotduarte** [reply]
You can check issues:
https://github.com/marcelotduarte/cx_Freeze/issues?q=is%3Aissue+is%3Aopen+scipy

**Reply by marcelotduarte** [reply]
@Venoob  Do you have success with this? If no, open an issue and put the traceback that I'll take a look an create a hook for this.

**Reply by Venoob** [reply]
hello,I got new error,  thanky you very much, I'll open an issue

**Reply by Venoob** [reply]
[https://github.com/marcelotduarte/cx_Freeze/issues/947](url)

**Reply by marcelotduarte** [reply]
The error with scipy.special.cython_special, do you resolved it with includes?

**Reply by Venoob** [reply]
I  finally solved the issue , with downgrading to python 3.7 . I Have a new problem but I think that this time It's my windows system who need to be updated .
